src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,75 @@
-export const Pagination = () => {};
+import React from 'react';
+import cn from 'classnames';
+import { getNumbers } from '../../utils';
+
+interface Props {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: React.Dispatch<number>;
+}
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage = 1,
```

Comment:
The default value assignment for 'currentPage' is unnecessary here since 'currentPage' is a required prop. If you intend to provide a default value, consider making the prop optional or handling the default assignment outside of the destructure.